Student/Student/Assignments/AssignmentDetails/AssignmentDetailsViewController.swift

```diff
@@ -407,7 +407,23 @@ class AssignmentDetailsViewController: ScreenViewTrackableViewController, Assign
         descriptionHeadingLabel?.text = quiz == nil
             ? NSLocalizedString("Description", bundle: .student, comment: "")
             : NSLocalizedString("Instructions", bundle: .student, comment: "")
-        webView.loadHTMLString(presenter?.assignmentDescription() ?? "", baseURL: baseURL)
+
+        let prefix = OfflineFolderPrefix.assignment.rawValue
+        let rootURL = URL.Directories.documents.appendingPathComponent(
+            URL.Paths.Offline.courseSectionFolder(
+                sessionId: env.currentSession?.uniqueID ?? "",
+                courseId: courseID,
+                sectionName: OfflineContainerPrefix.Assignments.rawValue
+            )
+        ).appendingPathComponent("\(prefix)-\(assignmentID)")
+        let offlinePath = rootURL.appendingPathComponent("body.html")
+        if offlineModeInteractor?.isNetworkOffline() == true && FileManager.default.fileExists(atPath: offlinePath.path) {
+            webView.loadFileURL(URL.Directories.documents, allowingReadAccessTo: URL.Directories.documents)
+            let rawHtmlValue = try? String(contentsOf: offlinePath, encoding: .utf8)
+            webView.loadHTMLString(rawHtmlValue ?? "", baseURL: rootURL)
```

Comment:
This seems to be used in a few places. Can we create a helper method for this in CoreWebView?